doc/refactoring.md

```diff
@@ -4,15 +4,15 @@ Some Smart Answer flows contain a mix of concerns: routing logic, policy/calcula
 
 These concerns should really be split. Routing belongs in the flow, policy/calculator logic belongs in the calculator and presentation belongs in the ERB templates. You might think of the flow as the Controller, the calculator as the Model and the ERB template as the View in an MVC architecture.
 
-We've refactored a small number of these Smart Answers and have a rough set of steps that we follow:
+We've refactored a number of these Smart Answers and have a rough set of steps that we follow:
 
 * Remove unused code from the flow, e.g. unused `save_input_as`, `precalculate` or `calculate` blocks. See commit [fe4014d7c007108f3daa07f1c5dd749f5de683a0](https://github.com/alphagov/smart-answers/commit/fe4014d7c007108f3daa07f1c5dd749f5de683a0) for an example.
 
 * Include `ActiveModel::Model` in the calculator so that it's easy to instantiate it with a number of attributes set. See commit [f78671a062339ae97abb5cb267b55536233316c9](https://github.com/alphagov/smart-answers/commit/f78671a062339ae97abb5cb267b55536233316c9) for an example.
 
-* Instantiate the calculator in a `next_node_calculation` block in the first question. See commit [6114c5fc55bfdaff52501e2a19dead35ebb42f13](https://github.com/alphagov/smart-answers/commit/6114c5fc55bfdaff52501e2a19dead35ebb42f13) for an example.
+* Instantiate the calculator in a `on_response` block in the first question. See commit [0df6fd7df9a00ec882edf249eeaaa68a291633c5](https://github.com/alphagov/smart-answers/commit/0df6fd7df9a00ec882edf249eeaaa68a291633c5) for an example.
```

Comment:
I've added a couple of inline example code blocks in an extra commit: 8762b6f023081311500cdcdf0bafe76b68c90573.